recipes/cgns/all/patches/fix_find_hdf5.patch

```diff
@@ -0,0 +1,112 @@
+The CGNS build uses the FindHDF5 module that locates the HDF5 libs by invoking the HDF5 compiler.
+In order to link to the correct libraries provided by conan, the whole logic is removed from the cmake script.
+
+--- a/source_subfolder/CMakeLists.txt	2020-02-14 16:16:46.000000000 +0100
++++ b/source_subfolder/CMakeLists.txt	2021-02-03 10:50:47.216808026 +0100
+@@ -194,104 +194,8 @@
+ option(CGNS_ENABLE_HDF5 "Enable or disable HDF5 interface" "OFF")
+ if (CGNS_ENABLE_HDF5)
+ 
+-  if (CGNS_BUILD_SHARED)
```

Comment:
Maybe it would be better to comment this big section rather than delete it?
Basically, use `if(FALSE)` trick.
It might make this patch more safe for future version.

Reply:
@Marks101 can you please apply this suggestion